bug(ci): failure in compilation of lua-resty-ldap in the CI

### Current Behavior

The following error is all around the CI. I tried to fix this assuming that compiling lua-resty-ldap would require a different version of the rust toolchain but that did not work. 🫤🫤🫤

```log
error[E0407]: method `decode_with_tag_and_constraints` is not a member of trait `crate::Decode`
 --> /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rasn-0.6.1/src/types/open.rs:5:65
  |
5 | #[derive(AsnType, Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Decode, Encode)]
  |                                                                 ^^^^^^ not a member of trait `crate::Decode`
  |
  = note: this error originates in the derive macro `Decode` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0407]: method `encode_with_tag_and_constraints` is not a member of trait `crate::Encode`
 --> /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rasn-0.6.1/src/types/open.rs:5:73
  |
5 | #[derive(AsnType, Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Decode, Encode)]
  |                                                                         ^^^^^^ not a member of trait `crate::Encode`
  |
  = note: this error originates in the derive macro `Encode` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0405]: cannot find trait `Choice` in module `crate::types`
 --> /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rasn-0.6.1/src/types/open.rs:5:10
  |
5 | #[derive(AsnType, Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Decode, Encode)]
  |          ^^^^^^^ not found in `crate::types`
  |
  = note: this error originates in the derive macro `AsnType` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0405]: cannot find trait `DecodeChoice` in module `crate::types`
 --> /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rasn-0.6.1/src/types/open.rs:5:65
  |
5 | #[derive(AsnType, Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Decode, Encode)]
  |                                                                 ^^^^^^ not found in `crate::types`
  |
  = note: this error originates in the derive macro `Decode` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0412]: cannot find type `Constraints` in module `crate::types`
 --> /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rasn-0.6.1/src/types/open.rs:5:65
  |
5 | #[derive(AsnType, Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Decode, Encode)]
  |                                                                 ^^^^^^ not found in `crate::types`
  |
  = note: this error originates in the derive macro `Decode` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0412]: cannot find type `Constraints` in module `crate::types`
 --> /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rasn-0.6.1/src/types/open.rs:5:73
  |
5 | #[derive(AsnType, Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Decode, Encode)]
  |                                                                         ^^^^^^ not found in `crate::types`
  |
  = note: this error originates in the derive macro `Encode` (in Nightly builds, run with -Z macro-backtrace for more info)
```

https://github.com/apache/apisix/actions/runs/5528944633/jobs/10086345094#step:6:1087
https://github.com/apache/apisix/actions/runs/5528937798/jobs/10086583852?pr=9811 

### Expected Behavior

_No response_

### Error Logs

_No response_

### Steps to Reproduce

NA

### Environment

- APISIX version (run `apisix version`):
- Operating system (run `uname -a`):
- OpenResty / Nginx version (run `openresty -V` or `nginx -V`):
- etcd version, if relevant (run `curl http://127.0.0.1:9090/v1/server_info`):
- APISIX Dashboard version, if relevant:
- Plugin runner version, for issues related to plugin runners:
- LuaRocks version, for installation issues (run `luarocks --version`):

--- a/ci/common.sh
+++ b/ci/common.sh
@@ -102,6 +102,9 @@ install_nodejs () {
 install_rust () {
     curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sudo sh -s -- -y
     source "$HOME/.cargo/env"
+    # 1.69.0 version required to compile lua-resty-ldap
+    rustup install 1.69.0
+    rustup default 1.69.0
 }
 
 set_coredns() {

